### DISCUSSION TITLE
Art Gallery Show - Awareness

### DISCUSSION THREAD

**Discussion by Moonaramoona** [WeAreMahsaAmini/Core#139] [Ideas]
To create a platform to show all the artworks that have been illustrated by known and unknown people. There are many beautiful and impressive drawings, sketches, and artistic posters that exist but all are diffused.  It would be helpful if there will be a platform where many people can see them in one place which can increase awareness plus it will motivate more local and foreign artists to engage and create content.

**Comment by sinahnyazdi** [comment]
That's a great idea, however it's doable on instagram and no need to build the platform from scratch

**Comment by Moonaramoona** [comment]
@sinahnyazdi It is doable but not efficient and effective. We don't have such a page on Instagram that collected all artworks. But the main reason we need to have a separate platform is that you can share the link increase awareness and engagement. 

**Disadvantages of Instagram**

- Doesn't let international artists involve as many people writing in Farsi
- The threat of copyright is so high
- Usable on mobile only and it looks so ugly on PC
- Many people don't have Instagram
- It won't motivate artists to engage or they may never get informed


**Advantages of Platform:**

- You will know the name of the artist by clicking on the artwork, it can link viewers either to their Instagram page or their websites, or any platform they want to show.
- You can engage ordinary people  by liking, commenting, and sharing their artwork on their social media pages
- Usable on web and mobile
- Official websites are going to talk about it and promote it. They just need to hyperlink the platform. but for Instagram they cant
- Highly increase awareness but also has a high potential to increase engagement by new artists
- The number of people who are going to engage is much more than celebrities and will be repeated.
- Artists is one of the best target audience because of their potential and their target audience but they can't engage in Instagram page and don't have a credit
- Later you can add 3D videos
- This campaign is very similar to Celebrity Wall. Need to be executed by a team and needs the team to engage artists
- Each artist can have its album.
- You can add features but with Instagram you cant
- In the end, you enjoy watching the artworks on large scale and willingly share them and talk about them
- It can be a part of "Wearemahsaamini" website
